Pod/Classes/BONChain.m

```diff
@@ -441,4 +441,9 @@ - (BONChainColor)textColor
     return self.color;
 }
 
+- (void)appendLink:(id<BONTextable>)link separator:(NSString *)separator
+{
+    [self appendLink:link separatorTextable:BONChain.new.string(separator)];
```

Comment:
#req I may have misled you before. I believe, to keep the original behavior, you need to use `self.string(separator)` here. If you change it, you'll see that some of your test changes result in failing tests.